apps/zotonic_mod_admin/priv/templates/_admin_overview_list.tpl

```diff
@@ -121,18 +129,24 @@ qcat
                 <input type="checkbox" value="{{ id }}" name="csel">
             </td>
             <td>
+                <span {% include "_language_attrs.tpl" %}>{{ id.title|default:id.short_title|striptags|default:_"<em>Untitled</em>" }}</span>
+                {% if id.is_protected %}
+                    &nbsp; <span class="fa fa-lock text-muted" title="{_ Protected, not deletable _}"></span>
+                {% endif %}
+
                 {% if id == 1 or id.is_a.meta or id.content_group_id.name == 'system_content_group' %}
-                    <span class="label label-warning pull-right" title="{_ This is system content. _}">
+                    <span class="label label-warning pull-right hidden-xs" title="{_ This is system content. _}">
                         {{ id.name|default:_"system content" }}
                     </span>
                 {% elseif id.name %}
-                    <span class="label label-default pull-right">
+                    <span class="label label-default pull-right hidden-xs">
                         {{ id.name }}
                     </span>
                 {% endif %}
-                <span {% include "_language_attrs.tpl" %}>{{ id.title|default:id.short_title|striptags|default:_"<em>Untitled</em>" }}</span>
-                {% if id.is_protected %}
-                    &nbsp; <span class="fa fa-lock text-muted" title="{_ Protected, not deletable _}"></span>
+                {% if id | is_a:"person" %}
```

Comment:
Logically anything can be a user.  Should we just check on username?   That will give a query for every row (am not sure if it is cached).

Reply:
Yes, probably not. That is why I limited it to person resources for now. I also don't think the username panel will appear in the admin for non-person resources.

Reply:
Maybe each resource - at OpenResearch they have non-person resources with a username.